common/presenters/table/object-to-table-head.js

```diff
@@ -1,17 +1,49 @@
 const { pickBy } = require('lodash')
+const queryString = require('query-string')
 
 const i18n = require('../../../config/i18n')
 
-function objectToTableHead({ head } = {}) {
-  if (!head) {
-    return undefined
+function buildUrl(head) {
+  const { query, key } = head
+  return queryString.stringify({
+    ...query,
+    sortBy: key,
+    sortDirection: (query && query.sortDirection) === 'asc' ? 'desc' : 'asc',
+  })
+}
+
+function buildWrapper(head) {
+  const label = i18n.t(head.html)
```

Comment:
This might read easier if you return the non-sortable option first. It also saves having to call `buildUrl` on columns that aren't sortable. Something like this:

```javascript
function buildSortableLink({ sortable, html, query, key }) {
  const label = i18n.t(head.html)

  if (!sortable) {
    return label
  }

  return `<a class="sortable-table__button" role="button" href="?${buildUrl(query, key)}">${label}</a>`
}
```